text/0092-logs-vision.md

```diff
@@ -0,0 +1,128 @@
+# OpenTelemetry Logs Vision
+
+The following are high-level items that define our long-term vision for 
+Logs support in OpenTelemetry project, what we aspire to achieve.
+
+This a vision document that reflects our current desires. It is not a commitment
+to implement everything precisely as listed. The primary purpose of this
+document is to ensure all contributors work in alignment. As our vision changes
+over time maintainers reserve the right to add, modify, and remove items from
+this document.
+
+(The content in this proposal is a starting point for discussion. I believe it
+is important for us to collectively come up with such vision. I am posting
+this PR largely to open the discussion. Comments, thoughts and opinions are
+highly welcome!)
+
+This document uses vocabulary introduced in https://github.com/open-telemetry/oteps/pull/91
+
+### First-class Citizen
+
+Logs are first-class citizen in observability, along with traces and metrics.
+We will aim to have best-in-class support for logs at OpenTelemetry.
+
+### Correlation
+
+OpenTelemetry will define how logs will be correlated with traces and metrics
+and how this correlation information will be stored.
+
+Correlation will work across 2 major dimensions:
+- To correlate telemetry emitted for the same request (also known as Request
+  or Trace Context Correlation),
+- To correlate telemetry emitted from the same source (also known as Resource
+  Context Correlation).
+
+### Logs Data Model
+
+We will design a Log Data model that will aim to correctly represent all types
+of logs and events. The purpose of the data model is to have a common
```

Comment:
We might need to clarify `events` here, is this a special type of `log`?

Reply:
Removed `event`, not important in this document.